client/App.jsx

```diff
@@ -73,15 +73,15 @@ class App extends React.Component {
   getCoreData() {
     axios.get(`/listings/${this.props.listingId}/booking/core`)
       .then((response) => {
-        const listing = response.data[0];
+        const listing = response.data.rows[0];
         this.setState({
-          costPerNight: listing.avg_cost_per_night,
+          costPerNight: JSON.parse(listing.avg_cost_per_night),
```

Comment:
are you parsing your data in the client side?

Reply:
Yeah. Some values are returned as strings but I need them to be number type